tools/validators/ontology_validator/README.md

```diff
@@ -45,11 +45,11 @@ The validator is Python-based and takes the following parameters:
 * `--modified-ontology-types` or `-m` **[Optional]**: An absolute or relative path to the modified files of the ontology.
 * `--interactive` or `-i` **[Optional]**: Enables interactive mode.
 
-The validator can be run as follows: `python3 validator.py -o=Users/foo/ontology/` or `python3 validator.py --original=Users/foo/ontology/`
+The validator can be run as follows: `python3 validator.py -o=Users/foo/ontology/yaml/resources` or `python3 validator.py --original=Users/foo/ontology/yaml/resources`
```

Comment:
I mean a user's ontology doesn't have to be under `ontology/yaml/resources`, that's just how we've done it. This still assumes that some poor soul out there has made their own ontology. For the sake of comprehension though, we could make this `path/to/original/ontology/`. Thoughts?

Reply:
Yes it does - we've had an MSI run into errors because there are folder trees in the `ontology` level that the validator is not happy about